src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -844,7 +866,7 @@ public String getGerritCmdBuildSuccessful() {
      * @param cmd the command
      * @see #getGerritCmdBuildSuccessful()
      */
-    public void setGerritVerifiedCmdBuildSuccessful(String cmd) {
+    public void setGerritCmdBuildSuccessful(String cmd) {
```

Comment:
These methods ware renamed to match respective getters. Otherwise, CasC do not recognize them as a getter/setter for the same property.

Reply:
Done